test/e2e/specs/book-event.js

```diff
@@ -438,4 +438,11 @@ describe('Book event page', () => {
     Booking.modifyButton.click();
     EventSessionsPage.sessionsHeader.waitForVisible();
   });
+
+  it('should not allow the user to continue in the booking flow when the event is full', () => {
+    DojoPage.openDojoWithLatLong(10, 89, 1);
+    DojoDetailsPage.name.waitForVisible();
+    expect(DojoDetailsPage.eventViewButtons(0).getText()).equals('FULL');
+    expect(DojoDetailsPage.eventViewButtons(0).isEnabled()).equals(false);
```

Comment:
Might be good to click the button and assert that we haven't changed page. The `isEnabled()` should be enough though...